e2e_test.go

```diff
@@ -484,139 +484,166 @@ func TestIPv6Lookup(t *testing.T) {
 }
 
 func TestLibp2pACMEE2E(t *testing.T) {
-	db := pebbleDB.NewMemoryStore()
-	logger := log.New(os.Stdout, "", 0)
-	ca := pebbleCA.New(logger, db, "", 0, 1, 0)
-	va := pebbleVA.New(logger, 0, 0, false, dnsServerAddress, db)
+	tests := []struct {
+		name       string
+		clientOpts []client.P2PForgeCertMgrOptions
+	}{
+		{
+			name:       "default opts",
+			clientOpts: []client.P2PForgeCertMgrOptions{},
+		},
+		{
+			name:       "explicit WithShortForgeAddrs(true)",
+			clientOpts: []client.P2PForgeCertMgrOptions{client.WithShortForgeAddrs(true)},
```

Comment:
@guillaumemichel I've added explicit e2e tests for all three states (implicit default, true, false)  and it seems to fail when short `/dnsX` addr is used (true):

```
--- FAIL: TestLibp2pACMEE2E (39.18s)
    --- PASS: TestLibp2pACMEE2E/default_opts (13.18s)
    --- FAIL: TestLibp2pACMEE2E/explicit_WithShortForgeAddrs(true) (12.99s)
    --- PASS: TestLibp2pACMEE2E/explicit_WithShortForgeAddrs(false) (13.00s)
```

- https://github.com/ipshipyard/p2p-forge/actions/runs/12778757594/job/35622173875?pr=40#step:13:363

I'm out this Wednesday but maybe you could take a look why it fails?